app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -99,50 +119,68 @@
     "Liechtenstein",
```

Comment:
Apparently there is also a Lao People's Democratic Republic, but we're missing that one.